plugins/woocommerce/src/Admin/Features/Settings/Init.php

```diff
@@ -172,8 +172,13 @@ public static function add_component_settings( $settings ) {
 			$settings['settingsScripts'][ $setting_page->get_id() ] = self::get_script_urls( $settings_scripts_handles );
 		}
 
-		$transformer              = new Transformer();
-		$settings['settingsData'] = $transformer->transform( $pages );
+		$transformer                       = new Transformer();
+		$settings['settingsData']['pages'] = $transformer->transform( $pages );
+		$start_hook_content                = $setting_pages[0]->get_custom_view( 'woocommerce_settings_start' );
```

Comment:
I'm not familiar with the hook. Just curious, does this mean the `woocommerce_settings_start` hook always returns the same content for all pages?

Reply:
> Just curious, does this mean the `woocommerce_settings_start` hook always returns the same content for all pages?

Yes, thats right. Its added before any content and does not change based on query parameters.

https://github.com/woocommerce/woocommerce/blob/0135e4943791d9ba8a6281c0f453d6b9754c0225/plugins/woocommerce/includes/admin/class-wc-admin-settings.php#L142